tools/dds/poc/h/poc-h.cpp

```diff
@@ -106,11 +109,17 @@ int main( int argc, char** argv ) try
         }
     }
 
+    LOG_DEBUG( "domain id: " << domain);
```

Comment:
This is already output when the participant comes online...

Reply:
OK will remove